### PR TITLE
Feature/new v5 hca file index

### PR DESCRIPTION
**Pull request by caaespin**
Modified the webservice to allow it to read from the new index structure. Fixed the summary and files endpoints. 

This is a rushed fix so cleaning should be done in a subsequent PR. The unit tests need to be fixed.

### CODE REVIEW

**Comment by hannes-ucsc** [comment]
Somehow this looks like it mixes in unrelated changes. Is the base of this branch correct?

**Comment by caaespin** [comment]
@hannes-ucsc I branched out of Jim's chalice-support branch.

File: app.py [hunk 1]
```diff
@@ -100,18 +100,21 @@ def get_data(file_id=None):
             filters['file']['fileId'] = {"is": [file_id]}
         # Create and instance of the ElasticTransformDump
         logger.info("Creating ElasticTransformDump object")
-        es_td = EsTd(es_domain=os.getenv("ES_DOMAIN", "elasticsearch1"),
```
Comment (hannes-ucsc) [inline app.py:103]: Confused by this.

File: chalicelib/responseobjects/elastic_request_builder.py [hunk 2]
```diff
@@ -52,9 +55,25 @@ def __init__(
             "Protocol must be 'http' or 'https'"
         self.logger.debug('ElasticSearch url: {}://{}:{}/'.format(
             es_protocol, es_domain, es_port))
-        self.es_client = Elasticsearch(
-            ['{}://{}:{}/'.format(es_protocol, es_domain, es_port)],
-            timeout=90)
+        if es_domain.endswith('.es.amazonaws.com'):
```
Comment (hannes-ucsc) [inline chalicelib/responseobjects/elastic_request_builder.py:58]: … by this

File: requirements.txt [hunk 3]
```diff
@@ -1,53 +1,12 @@
-alembic==0.8.10
```
Comment (hannes-ucsc) [inline requirements.txt:1]: … and all of these changes.

**Review by hannes-ucsc — APPROVED** [review]
This still seems to conflate changes of different nature but I'll leave it at that. We'll talk tomorrow.

**Comment by hannes-ucsc** [comment]
Merge at will.

**Pull request merged**